test/session_form_extensions.rb

```diff
@@ -219,6 +221,67 @@ def get_field!(id)
       get_field(id, :strict)
     end
 
+    # Find the field whose name ends in the given String or matches the given
```

Comment:
New style forms (using [Model Object Helpers](https://guides.rubyonrails.org/v5.2/form_helpers.html#dealing-with-model-objects-model-object-helpers) instead of `_tag` do not create `id`s. 
So I tweaked/hacked `test/session_form_extensions.rb` to accept pseudo-`name`s.

Reply:
hmmm. Looking into it...